data/repositories.toml

```diff
@@ -330,6 +330,7 @@ repositories = [
   'github.com/Kong/kong',
   'github.com/Kong/kuma',
   'github.com/krassowski/jupyterlab-lsp',
+  'github.com/krishdevdb/reseter.css'
```

Comment:
Can you please fix the syntax here? There's a missing comma that's breaking the TOML.